### PR TITLE
#44 Display build information

### PR DESCRIPTION
**Pull request by kislenko-artem**
Update RunREPL() to be able to display build information

Updates: #52

### CODE REVIEW

**Comment by kislenko-artem** [comment]
@corona10 ok. But how will you fill these  variables? go build will work anyway. if compilation will run not via makefile, variables will retain default value/

**Comment by corona10** [comment]
@kislenko-artem 
Please read the documentation which is linked on issue #52
https://goreleaser.com/environment/#using-the-main-version
I am not a go-releaser expert but on the documentation they say:
```
Default wise GoReleaser sets three ldflags:
main.version: Current Git tag (the v prefix is stripped) or the name of the snapshot, if you’re using the --snapshot flag
main.commit: Current git commit SHA
main.date: Date according RFC3339
```

**Comment by corona10** [comment]
Please fill the PR description if you possible.
e.g
````
Update RunREPL() to display build information

Updates: #52 
````

**Comment by kislenko-artem** [comment]
@corona10 unfortunately I can not fix description, I can edit only title. I removed makefile, sorry for my carelessness, I missed that project build via `goreleaser`

Now it is looked like this:
selection_845

File: version.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package main
```
Comment (corona10) [inline version.go]: Please add a license comments

````
// Copyright 2018 The go-python Authors.  All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.
````

File: main.go [hunk 2]
```diff
@@ -63,7 +63,7 @@ func main() {
 	args := flag.Args()
 	py.MustGetModule("sys").Globals["argv"] = pysys.MakeArgv(args)
 	if len(args) == 0 {
-		cli.RunREPL()
+		cli.RunREPL(version, commit, date)
```
Comment (corona10) [inline main.go]: @ncw 
Is there any reason not to move
```go
fmt.Printf("Python 3.4.0 (%s, %s)\n", commit, date)
fmt.Printf("[Gpython %s]\n", version)
fmt.Printf("- os/arch: %s/%s\n", runtime.GOOS, runtime.GOARCH)
fmt.Printf("- go version: %s\n", runtime.Version())
```
By this way, we can avoid to pass `version, commit, date` to exectute `RunREPL`

Comment (ncw) [reply main.go]: I guess that would probably be OK!

Comment (corona10) [reply main.go]: @kislenko-artem Please follow the comment I left! Thanks!

**Comment by corona10** [comment]
@kislenko-artem
Any progress on this PR?

cc @ncw

**Comment by kislenko-artem** [comment]
@corona10very very sorry, I thought my participation was ended. I added copyright information.

**Comment by corona10** [comment]
@kislenko-artem

Almost done,
Please follow the comment I left.
https://github.com/go-python/gpython/pull/52/files#r252264657

We don't want to pass paramters to RunREPL for the version information

File: main.go (hunk 2 above)
Comment (kislenko-artem) [reply main.go]: @corona10 sorry I can't figure out what I have to do. Output of version of Python was inside Repl function. It was just - fmt.Printf("Gpython 3.4.0\n") I added some attributes. Where output of version should be? before cli.RunREPL and inside block if len(args) == 0 ? Is it OK?

Comment (corona10) [reply main.go]: @kislenko-artem I mean move out the fmt.Printf from RunREPL,
and do it on main.go :)

File: repl/cli/cli.go [hunk 3]
```diff
@@ -7,14 +7,13 @@ package cli
 
 import (
 	"fmt"
+	"github.com/go-python/gpython/py"
```
Comment (corona10) [inline repl/cli/cli.go]:
```go
	"fmt"
	"io"
	"os"
	"os/user"
	"path/filepath"

	"github.com/go-python/gpython/py"
	"github.com/go-python/gpython/repl"
	"github.com/peterh/liner"
```

Comment (corona10) [reply repl/cli/cli.go]: "github.com/peterh/liner" what for?

**Comment by corona10** [comment]
@kislenko-artem

Thank you for your contribution!
Thanks a lot!

**Pull request merged**